src/components/common/modal/styles.js

```diff
@@ -0,0 +1,2 @@
+export const lowZIndex = 1
```

Comment:
I think we can add those values to the theme directly instead of using constants